### ISSUE TITLE
Suggest GHCup instead of Haskell Platform

### ISSUE THREAD

**Issue by konsumlamm** [learnyouahaskell/learnyouahaskell.github.io#16]
The [What you need to dive in](https://learnyouahaskell.github.io/introduction.html#what-you-need) section in the introduction currently recommends downloading the Haskell Platform. However, the Haskell Platform is [deprecated](https://www.haskell.org/platform/) and it is recommended to use [GHCup](https://www.haskell.org/ghcup/) instead, so the section should be updated.

**Comment by smith558** [comment]
Definitely! I myself use GHCup for my environment management too as most people probably do. I will look into updating this when I find a moment. Alternatively, feel free to open a pull request. 🙂
